app/assets/src/components/views/phylo_tree/PhyloTreeCreation.jsx

```diff
@@ -248,7 +248,7 @@ class PhyloTreeCreation extends React.Component {
   parsePhyloTreeData(phyloTreeData) {
     return phyloTreeData.map(row => ({
       name: row.name,
-      user: row.user.name,
+      user: row.user ? row.user.name : null,
```

Comment:
This works but I like the short `(row.user || {}).name`

Reply:
cool